test/README.md

```diff
@@ -5,6 +5,10 @@
 
     virtualenv .venv
     sudo pip install -r requirements.txt
+    
+    echo "deb https://dl.bintray.com/rackhd/debian trusty main" | sudo tee -a /etc/apt/sources.list.d/rackhd-trusty.list
```

Comment:
It didn't occur to me when I reviewed https://github.com/RackHD/on-http/pull/76, but I think we should be packaging this as a tarball or wheel that can be installed using pip so that the package can be installed in the virtualenv rather than in dist-packages.